src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      {isMarried && sex === 'm' ? (
+        <p className="Person__partner">
+          {`${partnerName} is my wife`}
+        </p>
+      ) : (
+        ''
+      )}
+      {sex === 'f' && (
+        <p className="Person__partner">{`${partnerName} is my husband`}</p>
+      )}
+      {!isMarried && <p className="Person__partner">I am not married</p>}
```

Comment:
I think you can simplify it like so:
```suggestion
    <p className="Person__partner">
      {isMarried ? (
        `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
      ) : (
        'I am not married'
      )}
    </p>
```